api-code-examples/rust/docs-join.rs

```diff
@@ -0,0 +1,18 @@
+import iroh
+import asyncio
+
+
+async def main():
+    # Create in memory iroh node
+    node = await iroh.Iroh.memory()
+    node_id = await node.node().node_id()
+    print(f"Started Iroh node: {node_id}")
+
+    # you'll need to get a ticket from somewhere, see the `doc share` command documentation
+    # for details. This ticket will fail to join, but is a valid ticket.
+    TICKET = "docaaa7qg6afc6zupqzfxmu5uuueaoei5zlye7a4ahhrfhvzjfrfewozgybl5kkl6u6fqcnjxvdkoihq3nbsqczxeulfsqvatb2qh3bwheoyahacitior2ha4z2f4xxk43fgewtcltemvzhaltjojxwqltomv2ho33snmxc6biajjeteswek4ambkabzpcfoajganyabbz2zplaaaaaaaaaagrjyvlqcjqdoaaioowl2ygi2likyov62rofk4asma3qacdtvs6whqsdbizopsefrrkx"
+
+    doc = await node.docs().join(TICKET)
+    print(f"Joined doc: {doc.id()}")
+
+asyncio.run(main())
```

Comment:
This file seems like it's pyhton, but stored as `docs-join.rs`, is that intentional?